bitwarden_license/test/Scim.IntegrationTest/Controllers/v2/GroupsControllerTests.cs

```diff
@@ -0,0 +1,601 @@
+﻿using System.Text.Json;
+using Bit.Scim.IntegrationTest.Factories;
+using Bit.Scim.Models;
+using Bit.Scim.Utilities;
+using Bit.Test.Common.Helpers;
+using Xunit;
+
+namespace Bit.Scim.IntegrationTest.Controllers.v2;
+
+public class GroupsControllerTests : IClassFixture<ScimApplicationFactory>, IAsyncLifetime
+{
+    private const int InitialGroupCount = 3;
+    private const int InitialGroupUsersCount = 2;
```

Comment:
Minor formatting nit, Rider is complaining that these don't follow the private member naming convention, e.g. `_initialGroupCount`